ArduPlane/ArduPlane.cpp

```diff
@@ -801,7 +801,8 @@ void Plane::update_navigation()
             
     case RTL:
         if (quadplane.available() && quadplane.rtl_mode == 1 &&
-            nav_controller->reached_loiter_target()) {
+            nav_controller->reached_loiter_target() &&
+            AP_HAL::millis() - last_mode_change_ms > 1000) {
```

Comment:
we want to give the RTL nav controller a chance to run and make a decision before we switch to QRTL. So we require the RTL last at least 1 second. Then we look at whether we've reached the loiter target, and if we have then we switch to QRTL